planet/System.c4g/LanguageUS.txt

```diff
@@ -769,7 +769,7 @@ IDS_MSG_CHATDISCLAIMER=Chat - Disclaimer
 IDS_MSG_CLEARTHEMESSAGEBOARD=Clear the message board.
 IDS_MSG_CLIENT=client
 IDS_MSG_CMD_ABORT_NOCOUNTDOWN=Not in countdown!
-IDS_MSG_CMD_HOSTONLY=Host only!
```

Comment:
IDS_MSG_CMD_HOSTONLY is used for more commands than only /start, so it should either stay the same (or better reword it to something more kind like "Only the host can do that.") or be separated from the other commands.